src/sentry/static/sentry/app/views/performance/constants.tsx

```diff
@@ -70,3 +70,12 @@ export const AXIS_OPTIONS: TooltipOption[] = [
     label: t('p99 Duration'),
   },
 ];
+
+export const PERCENTILE_NAMES = new Set(
+  AXIS_OPTIONS.map(option => option.value).reduce((values: string[], value: string) => {
+    if (/^p[0-9]{2}\(\)$/.test(value)) {
```

Comment:
The percentile aggregate functions all have an output type of `duration` you could use that instead of regexing on the name. This pattern will miss p100 if we ever end up adding it.